lib/authentication/SecureStorage/customCredentialManager.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2015-2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+function CustomCredentialManager(storage) {
```

Comment:
this object creates unnecessary layer of abstraction so I think we can get rid of this file. Also our internal configuration parameters (like host or credType) shouldn't leak to a manager supplied by user - instead we should have as simple API as possible, hence instead of `read(host, user, credType)` we should have something more like `read(key)`

Reply:
Fixed